.travis.yml

```diff
@@ -198,9 +198,6 @@ install:
               echo "$COMPONENTS" | parallel --gnu -j10% "tfold {} 'cd {} && $COMPOSER_UP && $PHPUNIT_X$LEGACY'"
           elif [[ $deps = low ]]; then
               echo "$COMPONENTS" | parallel --gnu -j10% "tfold {} 'cd {} && $COMPOSER_UP --prefer-lowest --prefer-stable && $PHPUNIT_X'" &&
-              # Test the PhpUnit bridge on PHP 5.3, using the original phpunit script
```

Comment:
I tried, but there is no way to make this work. Instead, I'll test the master phpunit-bridge on appveyor on branch 2.7